### DISCUSSION TITLE
Add netcat into the image

### DISCUSSION THREAD

**Discussion by samuel-deal-tisseo** [docker-library/rabbitmq#705] [General]
Hello.

I've read the issue #326 and I understand why there is no official health-check.
However, I think it netcat should be included inside the image to allow users to set a simple `nc -z localhost 5672` healthcheck when then want it without having to build a custom image.

Thank you

**Comment by lukebakken** [comment]
Hello, thanks for using RabbitMQ.

Docker makes it so easy to add `netcat` yourself that it will not be considered here. Only utilities necessary to run RabbitMQ are included in these images.

**Comment by michaelklishin** [comment]
```
nc -z localhost 5672
```

is not meaningfully different from

```
rabbitmq-diagnostics check_port_connectivity -q
```

and just like that health check, it has one major problem: the node may or may not be accessible at `localhost`, or from a remote node if you run `rabbitmq-diagnostics` command there.

Instead of gathering information from GitHub issues, I'd consult [the official doc guide on Monitoring and Health checks](https://www.rabbitmq.com/docs/monitoring#health-checks-role-in-monitoring).
